TaskService/TaskService.csproj

```diff
@@ -141,6 +144,7 @@
     <Reference Include="System.Web.WebPages.Razor, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\packages\Microsoft.AspNet.WebPages.3.2.7\lib\net45\System.Web.WebPages.Razor.dll</HintPath>
     </Reference>
+    <Reference Include="System.Windows.Forms" />
```

Comment:
@bgavrilMS 
I thought that we would avoid this with 4.25.0 ?

that does not seem right!